gremlin/README.md

```diff
@@ -22,6 +22,7 @@ their default values. See values.yaml for all available options.
 | `affinity`                             | Map of node/pod affinities                                     | `{}`                                                                        |
 | `gremlin.apparmor`                     | Apparmor profile to set for the Gremlin Daemon                 | `""` (When empty, no profile is set)                                        |
 | `gremlin.teamID`                       | Gremlin Team ID to authenticate with                           | `""`                                                                        |
+| `gremlin.clusterID`                   | Arbitrary string that uniquely identifies your cluster (e.g. `my-production-cluster`) | `""`                                                                        |
```

Comment:
This whole table needs to be updated. This specific value will move to `gremlin.secret.clusterID`